test/built-ins/Function/built-in-function-objects-that-are-not-identified-as-constructors.js

```diff
@@ -0,0 +1,1611 @@
+// Copyright (C) 2020 Rick Waldron. All rights reserved.
+// This code is governed by the license found in the LICENSE file.
+/*---
+esid: sec-ecmascript-function-objects
+description: >
+  Built-in function objects that are not identified as constructors do not implement [[Construct]]
+
+info: |
+  ECMAScript Function Objects
+
+  All ECMAScript function objects have the [[Call]] internal method defined here. ECMAScript functions that are also constructors in addition have the [[Construct]] internal method.
+features: [arrow-function, Array.prototype.flatMap, Array.prototype.values, Symbol, Symbol.iterator, ArrayBuffer, Atomics, TypedArray, SharedArrayBuffer, BigInt, DataView, FinalizationRegistry, generators, Map, Object.fromEntries, Object.is, Promise.allSettled, Promise.any, Reflect, Reflect.construct, Reflect.set, Reflect.setPrototypeOf, Symbol.match, Symbol.matchAll, Symbol.replace, Symbol.search, Symbol.split, Set, String.prototype.endsWith, String.prototype.matchAll, string-trimming, String.prototype.trimEnd, String.prototype.trimStart, WeakRef, WeakMap, WeakSet]
+---*/
+assert.throws(TypeError, () => {
+  new decodeURI('');
+}, '`new decodeURI("")` throws TypeError');
```

Comment:
this doesn't verify it lacks [[Construct]] - you'd need Proxy to do that https://esdiscuss.org/topic/isconstructor#content-11

Reply:
Agreed, asserting that `new` throws `TypeError` gives false-positives.
[`isConstructor`](https://github.com/tc39/test262/blob/main/harness/isConstructor.js) harness we already have is way more useful.
It helped cover two bugs related to missing/extra `[[Construct]]` internal method in JSC: [`Proxy.revocable`](https://bugs.webkit.org/show_bug.cgi?id=210959) and [`Symbol` constructor](https://bugs.webkit.org/show_bug.cgi?id=211050).

Reply:
You're right, and I think that helped me recenter myself on these tests. I'm going to do both.

Reply:
> Agreed, asserting that new throws TypeError gives false-positives.

Yes, and I took steps to prevent false positives, on a case by case basis. 

Either way, I've re-worked the entire approach to include both the isConstructor checks and asserting "`new ...` + preventions" throws.